app/components/properties-panel/properties-panel.component.html

```diff
@@ -55,7 +55,13 @@
 
                             <template [ngIf]="pair[1].subtype(Type.Node)">
                                 <label  class="node-label">
-                                    {{selectedEntity[group[1]].get(pair[0]).drawableProperties.get('Label')}}
+                                    <!-- TODO: Figure out a better way to allow nodes to sometimes be null
+                                        
+                                        Specifically, this doesn't have a value yet, and this breaks a lot of 
+                                        promises that say that there is always a start state, that is, we can't 
```

Comment:
Are we going to move start state to be a graph only property then? This leads to the question what's the best way to let the plugin define special drawing in the graph editor for a start state? Currently it looks like nodes and the graph have a start state property? If nodes have that property I don't think there's a good way in the type system to say one and only one needs to be a start state.